cachecontrol/controller.py

```diff
@@ -260,6 +265,16 @@ def cache_response(self, request, response, body=None,
 
         response_headers = CaseInsensitiveDict(response.headers)
 
+        # what time is it now and from the header
+
```

Comment:
Let's remove the extra whitespace here so it feels connected to the following code.